geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/ConverterInputFormat.scala

```diff
@@ -118,3 +143,36 @@ class ConverterRecordReader extends FileStreamRecordReader with LazyLogging {
     }
   }
 }
+
+object ConverterRecordReader {
+
+  var converter: SimpleFeatureConverter = null
+  var sft: SimpleFeatureType = null
+  var confStr: String = null
+  var filter: Option[Filter] = null
+  var retypedSpec: Option[String] = null
```

Comment:
is it ok to have these global to the JVM? I guess for a given job, they would only be used by that job?

Reply:
The general idea to these changes was to prevent making as much stuff as possible during distributed combine since things won't change inside the mapper and they'll all end up the same value anyway.

Reply:
could you add some comments so that future us remembers this?